vividus-plugin-lighthouse/src/main/java/org/vividus/lighthouse/LighthouseSteps.java

```diff
@@ -121,6 +129,28 @@ public void performLighthouseScan(ScanType scanType, String webPageUrl,
         }
     }
 
+    @SuppressWarnings("unchecked")
+    private ArrayMap<String, BigDecimal> getMetrics(LighthouseResultV5 result)
+    {
+        List<ArrayMap<String, BigDecimal>> items = (List<ArrayMap<String, BigDecimal>>) result.getAudits()
+                .get("metrics").getDetails().get("items");
+        ArrayMap<String, BigDecimal> metrics = ArrayMap.create();
```

Comment:
is `ArrayMap` required?

Reply:
why not? its memory-efficient and used by this API

Reply:
it's a kind of internals of Liighthouse SDK, it's better to use Java Collections:
- internals are not exposed
- easier maintenance since Java API is changed rarely